README.rst

```diff
@@ -117,12 +117,26 @@ Set it to ``False`` if you have C++ executable files that end with the ``*.py``
     [pytest]
     cpp_ignore_py_files = False
 
+cpp_prefix
+^^^^^^^^^^
+
+This option allows the usage of tolls or emulators that are used by invoking them on
+the console wrapping the test binary, like wine or qemu:
+
+.. code-block:: ini
+
+    [pytest]
+    cpp_prefix = wine
```

Comment:
```suggestion
    cpp_prefix = wine -exec
    
The prefix will be used both for listing tests and running them.    
```

I want to make it clear that we support passing options as well (do not know if wine supports some options we can use? used `-exec` just as an example).

Reply:
I made example with qemu because wine usually doesn't require arguments